homeassistant/components/abode/__init__.py

```diff
@@ -134,19 +134,19 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
 def setup_hass_services(hass: HomeAssistant) -> None:
     """Home Assistant services."""
 
-    def change_setting(call):
+    def change_setting(call: ServiceCall) -> None:
         """Change an Abode system setting."""
-        setting = call.data.get(ATTR_SETTING)
-        value = call.data.get(ATTR_VALUE)
+        setting = call.data[ATTR_SETTING]
+        value = call.data[ATTR_VALUE]
```

Comment:
The attributes are required by the schema.
https://github.com/home-assistant/core/blob/00a4b60f741967c6d93d2c49c2de5490264a356c/homeassistant/components/abode/__init__.py#L49-L51